src/components/pages/maintenance/jobDetails/jobDetails.js

```diff
@@ -46,10 +47,10 @@ export class JobDetails extends Component {
     //       A long term fix would be to normalize the job data in maintenance.js (similar to how Telemetry is handled there).
     //       When/if that happens, remove all use of refreshPending in the local state of this component.
     if ((
-        nextProps.match.params.id !== (this.state.selectedJob || {}).jobId
-        || this.state.timeIntervalChangePending
-        || this.state.refreshPending
-      ) && nextProps.jobs.length) {
+      nextProps.match.params.id !== (this.state.selectedJob || {}).jobId
```

Comment:
Alignment could be wrong. Also, if the above open brackets are in separate lines it will be clearer so visualize the block